components/aws/sagemaker/tests/unit_tests/tests/test_utils.py

```diff
@@ -66,3 +67,35 @@ def test_write_output_json(self):
                 mock_path("/tmp/test-output").write_text.assert_called_with(
                     json.dumps(case)
                 )
+
+    def test_assume_default_boto3_session(self):
+        with patch("common._utils.boto3", MagicMock()) as mock_boto3:
+            returned_session = _utils.get_boto3_session("us-east-1")
+
+        assert returned_session == mock_boto3
```

Comment:
shouldn't this be indented inside the "with" block

Reply:
As like with a for loop, the variable actually continues to exist outside the scope of that block. I would probably put it inside the `with` block, but other parts of the file don't stick to that convention.